### DISCUSSION TITLE
How can I send additional data in response json after successful JWT authentication

### DISCUSSION THREAD

**Discussion by eartisto-MP** [micronaut-projects/micronaut-core#9755] [Q&A]
I am using Micronaut 4.0.2.
After successful JWT authentication, I want to send some user info after usual accessToken, refreshToken, etc. I tried passing attributes to AuthenticationReponse.success method, but that doesn't seems to work.
```java
Map<String, Object> attributes = new HashMap<>();
attributes.put("subscription_status", isSubscribed);
emitter.next(AuthenticationResponse.success(verifiedToken.getUid(), attributes));
emitter.complete();
```
I have used this technique in micronaut 3.7.x. So I know it used to work.

Any help is much appreciated.
Thanks

**Comment by sdelamo** [comment]
do you have an example which reproduces what you want to do in Micronaut Framework 4 which used to work in Micronaut Framework 3 ? I am not aware this has changed.

**Reply by ashwan1** [reply]
Code where it doesn't work in uploaded at below url:
https://github.com/ashwan1/micronaut404-discussion-9755/blob/main/app.zip

I basically copied jwt guide and added attributes to it. But when I hit '/login', I don't get extra attributes back in response.
I don't have access to my previous code but I am pretty sure this used to work.
